src/Compilers/VisualBasic/Portable/SymbolDisplay/ObjectDisplay.vb

```diff
@@ -333,7 +333,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ObjectDisplay
             Return (SymbolDisplayPartKind.StringLiteral << 16) Or AscW(c)
         End Function
 
-        Private Function Identifier(c As Char) As Integer
+        Private Function FieldName(c As Char) As Integer
+            Return (SymbolDisplayPartKind.FieldName << 16) Or AscW(c)
+        End Function
```

Comment:
i'm not really sure what's going on here.

Reply:
I don't know why this code is so complex, but I'd prefer to keep it that way.